src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -658,8 +659,13 @@ def register_submission_to_objects_api(
             },
         }
 
+        record_data = render_to_json(options["content_json"], context)
         object_data = prepare_data_for_registration(
-            submission, context, options, object_mapping
+            submission=submission,
+            record_data=record_data,
+            objecttype=options["objecttype"],
+            objecttype_version=options["objecttype_version"],
+            object_mapping=object_mapping,
```

Comment:
I seem to remember that this feature was a temporary solution for Utrecht. Does that mean we don't need to provide the new mechanism for object creation in the ZGW registration @joeribekker ?